### DISCUSSION TITLE
How to set window to maximum desktop width and height?

### DISCUSSION THREAD

**Discussion by kishaningithub** [tauri-apps/tauri#3518] [Q&A] [answered]
I am using mac osx i would like to know to set the app window to max screen width and height via config?

```json
    "windows": [
      {
        "title": "Taurdi",
        "width": 800,
        "height": 600,
        "resizable": true,
        "fullscreen": false
      }
```

The `fullscreen` is not desirable because in max osx fullscreen mode creates a new os level workspace and put this as the only app. 

Removing the width and height attributes makes it to a default width and height (which i think is 800 * 600)

**Comment by xuchaoqian** [comment, marked as answer]
Try the following config:
    "windows": [
      {
        "title": "Taurdi",
        "width": 800,
        "height": 600,
        "resizable": true,
        "maximized": true
      }
